Makefile

```diff
@@ -146,7 +146,7 @@ clean:
 # Clean the Lisp cache, reindex local projects.
 clean-cache:
 	@echo "Deleting $(LISP_CACHE)"
-	sbcl --noinform --non-interactive \
+	$(QUICKLISP) --noinform --non-interactive \
```

Comment:
Non-fatal, but `$(QUICKLISP)` already includes `--noinform --non-interactive` via its use of `$(SBCL)`, so those options can be removed.